server/api/users/users.test.ts

```diff
@@ -39,4 +39,37 @@ describe('auth router', () => {
       });
     });
   });
+
+  describe('GET /:id', () => {
+    it('should return error if user.id is not found', async () => {
+      const res = await server.exec.get(`/api/users/${randNumber()}`);
```

Comment:
We don't need a `randNumber()` here, any static number should suffice.

It shouldn't be an issue here since the database should be empty, but it's good to be careful when using random values in tests as it can result in inconsistent/unpredictable results...